### PR TITLE
Add ReactHN

### PR DESCRIPTION
**Pull request by addyosmani**
URL: https://react-hn.appspot.com

Validated against Chrome team's [Lighthouse](https://github.com/GoogleChrome/lighthouse/) PWA tool

Score:
screen shot 2016-06-23 at 15 45 37

### CODE REVIEW

File: src/assets/images/reacthn.svg [hunk 1]
```diff
@@ -0,0 +1,10 @@
...
...AAAQIECBAgQIAAAQKJBQQmiYevdQIECBAgQIAAAQIECBAgQKBZQGDS7GKVAAECBAgQIECAAAECBAgQSCwgMEk8fK0TIECAAAECBAgQIECAAAECzQICk2YXqwQIECBAgAABAgQIECBAgEBiAYFJ4uFrnQABAgQIECBAgAABAgQIEGgWEJg0u1glQIAAAQIECBAgQIAAAQIEEgsITBIPX+sECBAgQIAAAQIECBAgQIBAs4DApNnFKgECBAgQIECAAAECBAgQIJBYQGCSePhaJ0CAAAECBAgQIECAAAECBJoFBCbNLlYJECBAgAABAgQIECBAgACBxAICk8TD1zoBAgQIECBAgAABAgQIECDQLCAwaXaxSoAAAQIECBAgQIAAAQIECCQWWNV572fUnR/hAAIECBAgQIAAAQIECBAgQIBAmwLeYdKmpr0IECBAgAABAgQIECBAgACBUQgITEYxRk0QIECAAAECBAgQIECAAAECbQoITNrUtBcBAgQIECBAgAABAgQIECAwCgGBySjGqAkCBAgQIECAAAECBAgQIECgTQGBSZua9iJAgAABAgQIECBAgAABAgRGISAwGcUYNUGAAAECBAgQIECAAAECBAi0KSAwaVPTXgQIECBAgAABAgQIECBAgMAoBAQmoxijJggQIECAAAECBAgQIECAAIE2BQQmbWraiwABAgQIECBAgAABAgQIEBiFgMBkFGPUBAECBAgQIECAAAECBAgQINCmgMCkTU17ESBAgAABAgQIECBAgAABAqMQEJiMYoyaIECAAAECBAgQIECAAAECBNoUEJi0qWkvAgQIECBAgAABAgQIECBAYBQCApNRjFETBAgQIECAAAECBAgQIECAQJsCApM2Ne1FgAABAgQIECBAgAABAgQIjEJAYDKKMWqCAAECBAgQIECAAAECBAgQaFNAYNKmpr0IECBAgAABAgQIECBAgACBUQgITEYxRk0QIECAAAECBAgQIECAAAECbQoITNrUtBcBAgQIECBAgAABAgQIECAwCgGBySjGqAkCBAgQIECAAAECBAgQIECgTQGBSZua9iJAgAABAgQIECBAgAABAgRGISAwGcUYNUGAAAECBAgQIECAAAECBAi0KSAwaVPTXgQIECBAgAABAgQIECBAgMAoBP4/sdh8XCm4MrwAAAAASUVORK5CYII="></image>
```
Comment (pepelsbey) [inline src/assets/images/reacthn.svg]: I woudn’t call it SVG ;) `data:image/png;base64,iVBOR…`

Do you have a vector version of this logo?

Comment (pepelsbey) [inline src/assets/images/reacthn.svg]: [Found one](https://github.com/insin/react-hn/blob/master/public/img/safari-pinned-tab.svg) with actual curves, could you please update?

**Comment by addyosmani** [comment]
@pepelsbey Thanks for catching that. Updated:

screen shot 2016-06-24 at 12 01 30

**Comment by pepelsbey** [comment]
Thank you for suggestion! Optimized logo a bit in 8d0fa03, in case you’ll ever want to use it again.

**Pull request merged**